source/tutorials/sample-classifier.rst

```diff
@@ -0,0 +1,111 @@
+Predicting sample metadata categories with q2-sample-classifier
+===============================================================
+
+.. note:: This guide assumes you have installed QIIME 2 using one of the procedures in the :doc:`install documents <../install/index>`.
+
+This tutorial will demonstrate how to use ``q2-sample-classifier`` to predict sample metadata values. Supervised learning methods predict sample data (e.g., metadata values) as a function of other sample data (e.g., microbiota composition). The predicted targets may be discrete sample classes (for classification problems) or continuous values (for regression problems). Any other data may be used as predictive features, but for the purposes of q2-sample-classifier this will most commonly be microbial sequence variant, operational taxonomic unit (OTU), or taxonomic composition. However, any features contained in a feature table may be used — for non-microbial data, just `convert your observation tables to biom format`_ and `import to qiime2`_.
+
+We will download and create several files, so first create a working directory.
+
+.. command-block::
+   :no-exec:
+
+   mkdir sample-classifier-tutorial
+   cd sample-classifier-tutorial
+
+Predicting categorical sample data
+----------------------------------
+
+Supervised learning classifiers predict the categorical metadata classes of unlabled samples by learning the composition of labeled training samples. For example, we may use a classifier to diagnose or predict disease susceptibility based on stool microbiome composition, or predict sample type as a function of the sequence variants, microbial taxa, or metabolites detected in a sample. In this tutorial, we will use the `moving pictures tutorial data`_ to train a classifier that predicts the body site from which a sample was collected. Download the feature table and sample metadata with the following links:
+
+.. download::
+   :url: https://data.qiime2.org/2017.7/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: moving-pictures-sample-metadata.tsv
+
+.. download::
+   :url: https://docs.qiime2.org/2017.7/data/tutorials/moving-pictures/table.qza
+   :saveas: moving-pictures-table.qza
+
+Next, we will attempt to predict which body site each sample originated from based on its microbial composition.
+
+.. command-block::
+
+   qiime sample-classifier classify-samples \
+    --i-table moving-pictures-table.qza \
```

Comment:
This is pretty minor, but can you make sure the indented lines are 2 spaces (here it is 1), that way it is consistent with most (all?) of the other tutorials?